libs/etterlatte-trygdetid-model/src/main/kotlin/Trygdeavtale.kt

```diff
@@ -10,5 +11,12 @@ data class Trygdeavtale(
     val avtaleKode: String,
     val avtaleDatoKode: String?,
     val avtaleKriteriaKode: String?,
+    val personKrets: JaNei?,
```

Comment:
Samme som over, trenger alt å være optional?

Reply:
Jeg skal lage en oppgave for å se på det sammen med andre forbedringer på trygdeavtale